src/core_modules/capture-core/reducers/descriptions/workingLists/workingLists.reducerDescription.js

```diff
@@ -461,16 +460,27 @@ export const workingListsContextDesc = createReducerDescription({
     The meaning is slightly changed though, having a context now implies that a request for events was done for this context,
     not that events was successfully retrieved for this context.
     */
```

Comment:
Is this comment still above the right reducer case?

Reply:
Good catch. Rephrased a bit, because it applies to both reducer cases here.

Reply:
Sorry I think I didnt explain myself well. 

I think that now the comment is above an action called `TEMPLATES_FETCH` while as far as I understand the comments refers to the action with name `LIST_VIEW_INIT`. 

If this comment applies to the whole reducer perhaps then the best place to have it is above the line 
```
export const workingListsContextDesc = createReducerDescription({
```